initialize.js

```diff
@@ -12,6 +12,16 @@ for (const key in process.env) {
   }
 }
 
+// The stellar-sdk Client requires (for now) a defined public key. These are
+// the Genesis accounts for each of the "typical" networks, and should work as
+// a valid, funded network account.
+const GENESIS_ACCOUNTS = {
+  public: 'GAAZI4TCR3TY5OJHCTJC2A4QSY6CJWJH5IAJTGKIN2ER7LBNVKOCCWN7',
+  testnet: 'GBRPYHIL2CI3FNQ4BXLFMNDLFJUNPU2HY3ZMFSHONUCEOASW7QC7OX2H',
+  futurenet: 'GADNDFP7HM3KFVHOQBBJDBGRONMKQVUYKXI6OYNDMS2ZIK7L6HA3F2RF',
+  standalone: 'GBZXN7PIRZGNMHGA7MUUUF4GWPY5AYPV6LY4UV2GL6VJGIQRXFDNMADI',
+}
```

Comment:
Since the `ContractClient` constructor in the SDK requires a funded account's `publicKey` in its constructor, the genesis accounts seemed like a simple and reliable public key to use. It was aimed to be a workaround until stellar/js-stellar-sdk#941 gets rolled into a release.

Reply:
Ah got it so this will disappear when that PR is merged? Sgtm.

Reply:
That's the idea